include/Smarty/Smarty_Compiler.class.php

```diff
@@ -1024,7 +1024,8 @@ public function _compile_include_tag($tag_args)
             "\$_smarty_tpl_vars = \$this->_tpl_vars;\n";
 
 
-        $_params = "array('smarty_include_tpl_file' => " . $include_file . ", 'smarty_include_vars' => array(".implode(',', (array)$arg_list)."))";
+        $_params = "array('smarty_include_tpl_file' => " . $include_file . ", 'smarty_include_vars' => array(".implode(',',
+                $arg_list)."))";
```

Comment:
I don't think you should mess with the Smarty files (plugins are fine to mess with), since these are vendored from the Smarty package and any changes would cause problems when moving Smarty to the actual Composer package: #6719